code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -360,6 +360,12 @@
 		to_chat(L, "<span class='italics'>You hear a voice in your head saying: </span><span class='abductor'>[message]</span>")
 		to_chat(user, "<span class='notice'>You send the message to your target.</span>")
 		log_talk(user,"[key_name(user)] sent an abductor mind message to [L]/[L.ckey]: '[message]'", LOGSAY)
+		for(var/ded in GLOB.dead_mob_list)
+			if(!isobserver(ded))
```

Comment:
use deadchat_broadcast you chumbis

Reply:
With the second arg being `user` (you'll still have to add the follow link to `L` however)